modules/mirror-registry-release-notes.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-release-notes_{context}"]
+= Mirror registry for Red Hat OpenShift release notes
+
+The _mirror registry for Red Hat OpenShift_ is a small and streamlined container registry that you can use as a target for mirroring the required container images of {product-title} for disconnected installations.
+
+These release notes track the development of the _mirror registry for Red Hat Openshift_ in {product-title}.
+
+For an overview of the _mirror registry for Red Hat OpenShift_, see xref:../../installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-flags_installing-mirroring-creating-registry[Creating a mirror registry with mirror registry for Red Hat OpenShift].
+
+
+== Mirror registry for Red Hat OpenShift 1.2.0
```

Comment:
Do these various sections headings need an anchor ID?
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings